src/Blocks/continuous.jl

```diff
@@ -12,11 +12,15 @@ Initial value of integrator state ``x`` can be set with `x`
 # Parameters:
 
   - `k`: Gain of integrator
+
+# Unknowns:
+
+  - `x`: Unknown of Integrator (with initial guess of 0.0)
 """
 @mtkmodel Integrator begin
     @extend u, y = siso = SISO()
     @variables begin
-        x(t) = 0.0, [description = "State of Integrator"]
+        x(t), [description = "Unknown of Integrator", guess = 0.0]
```

Comment:
This instance of "state" should not be translated to "unknown", state is indeed correct here

Reply:
Resolved here: https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/274/commits/eae592ba86e9de11fa575f005d3c13b526a80c99#diff-2c628e949fda2edfd7af3ee31fbf986535a6b1e959d92d03be9ca070a4b633aeR23